x-pack/legacy/plugins/apm/public/components/app/Settings/AgentConfigurations/AddEditFlyout/ServiceSection.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiTitle, EuiSpacer, EuiFormRow } from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { SelectWithPlaceholder } from '../../../../shared/SelectWithPlaceholder';
+import { useFetcher } from '../../../../../hooks/useFetcher';
+import { ENVIRONMENT_NOT_DEFINED } from '../../../../../../common/environment_filter_values';
+import { Config } from '../index';
+import { callApmApi } from '../../../../../services/rest/callApmApi';
+const t = (id: string, defaultMessage: string) =>
+  i18n.translate(`xpack.apm.settings.agentConf.flyOut.serviceSection.${id}`, {
+    defaultMessage
+  });
```

Comment:
The i18n strings were getting long and starting to be a pain on the eyes. I played around with some different formats and I'm pretty pleased with this. Not saying we should do it everywhere but I think it's worth experimenting with. A couple of goals:

- the resulting call `t('key', 'label')` should be as short and concise as possible
- when renaming a file it should be easy to rename all i18n strings simultaneously
- the wrapper should not be too abstracted away from the original

Reply:
I'm assuming this breaks the i18n check?

Reply:
Maybe we can stop using a hierarchy for our labels. Instead of using `xpack.apm.settings.agentConfig.flyout.serviceSection.deleteConfigSucceededTitle`, we can use `xpack.apm.deleteAgentConfigSucceededTitle`?

Reply:
> I'm assuming this breaks the i18n check?

Hmm.. I do get this error when running the i18n check:
```
   ✖ x-pack/legacy/plugins/apm
     → EISDIR: illegal operation on a directory, read
```

Any idea how to fix?

> Maybe we can stop using a hierarchy for our labels. Instead of using xpack.apm.settings.agentConfig.flyout.serviceSection.deleteConfigSucceededTitle, we can use xpack.apm.deleteAgentConfigSucceededTitle

That would still be something like:
```ts
i18n.translate('xpack.apm.deleteAgentConfigSucceededTitle', {
  defaultMessage: 'Delete configuration'
}),
```

vs

```ts
t('deleteAgentConfigSucceededTitle', 'Delete configuration'),
```

Ofc if we can't make the later work then there's no way around it, but after having played around with it, it's a joy to have strings as one-liners again, and it my eyes slowly start reading the code again (instead of just ignoring all the stuff my brain perceives as noise)